xslt/Image.xsl

```diff
@@ -14,24 +15,42 @@
         </xsl:element>
     </xsl:template>
 
+    <xsl:variable name="img" select="/ImageDocument/Metadata/Information/Image"/>
+    <xsl:variable name="chs" select="/ImageDocument/Metadata/DisplaySetting/Channels"/>
```

Comment:
More of a curiosity, but, potentially an additional problem: I'm (obviously) in no way an expert in the CZI spec, and I know this is a reuse of the existing path, but, in my reading of the spec, this doesn't seem to me like the obvious place from which to access Channels.
The spec reads (pg. 54 in my version):
> The main use of DisplaySetting is to parametrize the GUI elements used in image visualization.

I suppose my upfront concern is that this could that imply that if no display setting for one or more channels was set through the GUI, that channel won't be found in `DisplaySettings`. Though, this concern/suspicion isn't backed up by anything.

Do you know if there's a record of why `Channels` are accessed through that path? Are the display attributes accessed (like, DyeName, or Color)?

Purely from the spec-standpoint, I would think the obvious path would be to go looking for Channels in `/ImageDocument/Metadata/Information/Image/Dimensions/Channels`.

Reply:
Fair point, and I'm honestly not sure why `DisplaySettings/Channels` is used instead of just `Dimensions/Channels`. A further interesting note is that I don't think any of the info from `Dimensions/Channels` is actually used. [See here in Channels.xsl](https://github.com/AllenCellModeling/czi-to-ome-xslt/blob/main/xslt/Channels.xsl#L111). @heeler, do you have any recollection of why it works this way?

I'll spend a few mins seeing what happens if I just directly use `Dimensions/Channels`.

Reply:
My bet is that DisplaySettings is info that ZEN uses to load and display the image / channel. What color to put on the channel, the range, etc. Whereas the Dimensions is where the actual channel data is located.

OME on the other hand I believe shoves all of that display and "actual" data into the same block. Hence why I think we use both?

Reply:
I'm pretty sure I can just use `Dimensions/Channels`. There are a couple things we currently pull from `DisplaySettings/Channels`, but the elements seem to be on both of these. Will try making the switch as part of this PR.

Reply:
@JacksonMaxfield that would make sense, but both places contain almost exactly the same data. Odd 🤷 Will certainly simplify the code a bit to stick with one.

Reply:
All right @GabeMedrash, let me know if this looks better now.

Reply:
Looks really good to me, from what I understand of it! Nice work